objects/ArkhamSCE350-1242024-Page1.2d0dbb.json

```diff
@@ -0,0 +1,182 @@
+{
+  "AltLookAngle": {
+    "x": 0,
+    "y": 0,
+    "z": 0
+  },
+  "Autoraise": true,
+  "ColorDiffuse": {
+    "b": 1,
+    "g": 1,
+    "r": 1
+  },
+  "Description": "Thanks for downloading Arkham SCE 3.5.0!\n\n- Added all officially-previewed Feast of Hemlock Vale player cards!\n- Added Parallel Monterey Jack!\n- The options panel has many new settings available.\n- Added new scripting to Empirical Hypothesis, accessible via right-clicking the card.\n",
+  "DragSelectable": true,
+  "GMNotes": "",
+  "GUID": "2d0dbb",
+  "Grid": true,
+  "GridProjection": false,
+  "Hands": false,
+  "HideWhenFaceDown": false,
+  "IgnoreFoW": false,
+  "LayoutGroupSortIndex": 0,
+  "Locked": false,
+  "LuaScript": "",
+  "LuaScriptState": "",
+  "MeasureMovement": false,
+  "Name": "Notecard",
+  "Nickname": "Arkham SCE 3.5.0 - 1/24/2024 - Page 1",
+  "Snap": true,
+  "States": {
+    "2": {
+      "AltLookAngle": {
+        "x": 0,
+        "y": 0,
+        "z": 0
+      },
+      "Autoraise": true,
+      "ColorDiffuse": {
+        "b": 1,
+        "g": 1,
+        "r": 1
+      },
+      "Description": "- The Deck Importer now supports custom cards! Requires using the Additional Player Cards bag and the nearby ArkhamDB instruction generator. Only works with cards with Zoop metadata.\n- A visual reference for the numpad hotkeys has been added near the Rules Reference.\n- Added a reshuffle button under the encounter card discard.",
```

Comment:
Some corrections:
- It works for any cards with an `id` in the metadata or a `ZoopGuid`
- the numpad reference was already in the last release

Reply:
Just pushed fixes for these, thanks